### DISCUSSION TITLE
mpv not launching on my older laptop

### DISCUSSION THREAD

**Discussion by skotarek** [mpv-player/mpv#14669] [Q&A] [answered]
I just tried launching my mpv installation on a different device
what I get when trying to launch mpv.exe is a WER\ReportArchive error like this:

>Version=1
EventType=APPCRASH
EventTime=133678700568623723
ReportType=2
Consent=1
UploadTime=133678700571956381
ReportStatus=268435456
ReportIdentifier=813b8979-b66c-49f5-8768-73a1612f31bc
IntegratorReportIdentifier=fe35ee86-6d08-4376-b520-5cdb2a64b4ac
Wow64Host=34404
NsAppName=mpv.exe
OriginalFilename=mpv.exe
AppSessionGuid=0000122c-0001-002c-bd1c-24e511ecda01
TargetAppId=W:000625f0350075d7b1f75c0ea39041a3600300000000!0000aa2df530adb1689bb2c6b318e7c2832dd8586d68!mpv.exe
TargetAppVer=2024//06//09:00:05:44!6743133!mpv.exe
BootId=4294967295
TargetAsId=2336
IsFatal=1
EtwNonCollectReason=1
Response.BucketId=e39c1e8e0261da49f97c6971dc4ac625
Response.BucketTable=4
Response.LegacyBucketId=1836458685808428581
Response.type=4
Sig[0].Name=Application Name
Sig[0].Value=mpv.exe
Sig[1].Name=Application Version
Sig[1].Value=2.0.0.0
Sig[2].Name=Application Timestamp
Sig[2].Value=6664f1d8
Sig[3].Name=Fault Module Name
Sig[3].Value=mpv.exe
Sig[4].Name=Fault Module Version
Sig[4].Value=2.0.0.0
Sig[5].Name=Fault Module Timestamp
Sig[5].Value=6664f1d8
Sig[6].Name=Exception Code
Sig[6].Value=c000001d
Sig[7].Name=Exception Offset
Sig[7].Value=00000000002cc83e
DynamicSig[1].Name=OS Version
DynamicSig[1].Value=10.0.19044.2.0.0.256.191
DynamicSig[2].Name=Locale ID
DynamicSig[2].Value=3072
DynamicSig[22].Name=Additional Information 1
DynamicSig[22].Value=5282
DynamicSig[23].Name=Additional Information 2
DynamicSig[23].Value=5282dcea7c5742e4e499384e4ab306f8
DynamicSig[24].Name=Additional Information 3
DynamicSig[24].Value=77c1
DynamicSig[25].Name=Additional Information 4
DynamicSig[25].Value=77c11134ff11bcc7b9226b3c1103eaf0
UI[2]=C:\Users\Master\Desktop\Apps\MPV\mpv.exe
LoadedModule[0]=C:\Users\Master\Desktop\Apps\MPV\mpv.exe
LoadedModule[1]=C:\Windows\SYSTEM32\ntdll.dll
LoadedModule[2]=C:\Windows\System32\KERNEL32.DLL
LoadedModule[3]=C:\Windows\System32\KERNELBASE.dll
LoadedModule[4]=C:\Windows\System32\GDI32.dll
LoadedModule[5]=C:\Windows\System32\win32u.dll
LoadedModule[6]=C:\Windows\System32\gdi32full.dll
LoadedModule[7]=C:\Windows\System32\msvcp_win.dll
LoadedModule[8]=C:\Windows\System32\ucrtbase.dll
LoadedModule[9]=C:\Windows\System32\USER32.dll
LoadedModule[10]=C:\Windows\System32\ADVAPI32.dll
LoadedModule[11]=C:\Windows\System32\msvcrt.dll
LoadedModule[12]=C:\Windows\System32\sechost.dll
LoadedModule[13]=C:\Windows\System32\RPCRT4.dll
LoadedModule[14]=C:\Windows\System32\bcrypt.dll
LoadedModule[15]=C:\Windows\System32\ole32.dll
LoadedModule[16]=C:\Windows\System32\combase.dll
LoadedModule[17]=C:\Windows\System32\SHELL32.dll
LoadedModule[18]=C:\Windows\System32\IMM32.dll
LoadedModule[19]=C:\Windows\System32\SHLWAPI.dll
LoadedModule[20]=C:\Windows\System32\WS2_32.dll
LoadedModule[21]=C:\Windows\SYSTEM32\VERSION.dll
LoadedModule[22]=C:\Windows\System32\OLEAUT32.dll
LoadedModule[23]=C:\Windows\System32\SETUPAPI.dll
LoadedModule[24]=C:\Windows\SYSTEM32\AVRT.dll
LoadedModule[25]=C:\Windows\System32\cfgmgr32.dll
LoadedModule[26]=C:\Windows\System32\CRYPT32.dll
LoadedModule[27]=C:\Windows\SYSTEM32\dwmapi.dll
LoadedModule[28]=C:\Windows\SYSTEM32\UxTheme.dll
LoadedModule[29]=C:\Windows\SYSTEM32\OPENGL32.dll
LoadedModule[30]=C:\Windows\SYSTEM32\AVICAP32.dll
LoadedModule[31]=C:\Windows\SYSTEM32\WINMM.dll
LoadedModule[32]=C:\Windows\SYSTEM32\USERENV.dll
LoadedModule[33]=C:\Windows\SYSTEM32\IPHLPAPI.DLL
LoadedModule[34]=C:\Windows\SYSTEM32\d2d1.dll
LoadedModule[35]=C:\Windows\SYSTEM32\DWrite.dll
LoadedModule[36]=C:\Windows\SYSTEM32\GLU32.dll
LoadedModule[37]=C:\Windows\SYSTEM32\MSVFW32.dll
LoadedModule[38]=C:\Windows\SYSTEM32\CRYPTBASE.DLL
LoadedModule[39]=C:\Windows\WinSxS\amd64_microsoft.windows.common-controls_6595b64144ccf1df_5.82.19041.4355_none_792ecafd2442822f\COMCTL32.dll
LoadedModule[40]=C:\Windows\System32\bcryptPrimitives.dll
State[0].Key=Transport.DoneStage1
State[0].Value=1
OsInfo[0].Key=vermaj
OsInfo[0].Value=10
OsInfo[1].Key=vermin
OsInfo[1].Value=0
OsInfo[2].Key=verbld
OsInfo[2].Value=19044
OsInfo[3].Key=ubr
OsInfo[3].Value=4651
OsInfo[4].Key=versp
OsInfo[4].Value=0
OsInfo[5].Key=arch
OsInfo[5].Value=9
OsInfo[6].Key=lcid
OsInfo[6].Value=1033
OsInfo[7].Key=geoid
OsInfo[7].Value=191
OsInfo[8].Key=sku
OsInfo[8].Value=191
OsInfo[9].Key=domain
OsInfo[9].Value=0
OsInfo[10].Key=prodsuite
OsInfo[10].Value=256
OsInfo[11].Key=ntprodtype
OsInfo[11].Value=1
OsInfo[12].Key=platid
OsInfo[12].Value=10
OsInfo[13].Key=sr
OsInfo[13].Value=0
OsInfo[14].Key=tmsi
OsInfo[14].Value=222794965
OsInfo[15].Key=osinsty
OsInfo[15].Value=1
OsInfo[16].Key=iever
OsInfo[16].Value=11.3636.19041.0-11.0.1000
OsInfo[17].Key=portos
OsInfo[17].Value=0
OsInfo[18].Key=ram
OsInfo[18].Value=3988
OsInfo[19].Key=svolsz
OsInfo[19].Value=232
OsInfo[20].Key=wimbt
OsInfo[20].Value=0
OsInfo[21].Key=blddt
OsInfo[21].Value=191206
OsInfo[22].Key=bldtm
OsInfo[22].Value=1406
OsInfo[23].Key=bldbrch
OsInfo[23].Value=vb_release
OsInfo[24].Key=bldchk
OsInfo[24].Value=0
OsInfo[25].Key=wpvermaj
OsInfo[25].Value=0
OsInfo[26].Key=wpvermin
OsInfo[26].Value=0
OsInfo[27].Key=wpbuildmaj
OsInfo[27].Value=0
OsInfo[28].Key=wpbuildmin
OsInfo[28].Value=0
OsInfo[29].Key=osver
OsInfo[29].Value=10.0.19041.4651.amd64fre.vb_release.191206-1406
OsInfo[30].Key=buildflightid
OsInfo[31].Key=edition
OsInfo[31].Value=IoTEnterpriseS
OsInfo[32].Key=ring
OsInfo[32].Value=Retail
OsInfo[33].Key=expid
OsInfo[33].Value=MD:283BAEF,ME:2E3922F,ME:2E30985,MD:2E30A2A
OsInfo[34].Key=fconid
OsInfo[35].Key=containerid
OsInfo[36].Key=containertype
OsInfo[37].Key=edu
OsInfo[37].Value=0
FriendlyEventName=Stopped working
ConsentKey=APPCRASH
AppName=mpv
AppPath=C:\Users\Master\Desktop\Apps\MPV\mpv.exe
NsPartner=windows
NsGroup=windows8
ApplicationIdentity=39DFBF96E9C13D6E49FFB6FBFB4B57AC
MetadataHash=1711664552

naturally, running `mpv --version` or `mpv.exe --version` does nothing either
I additionally tried removing all of my scripts and some configs too but that also didn't help
I'm not too sure what could be the issue here... is it that mpv has some requirements I'm not aware of?

**Comment by kasper93** [comment, marked as answer]
Download non-v3 version of mpv build.

**Reply by skotarek** [reply]
> Download non-v3 version of mpv build.

what's the difference?

**Reply by llyyr** [reply]
https://en.wikipedia.org/wiki/X86-64#Microarchitecture_levels

There's no difference for mpv, and I don't even know why the v3 build exists except to cause confusion among users

**Reply by skotarek** [reply]
okay so using the non-v3 version fixed the crash although I still had other problems and had to remove `gpu-api=vulkan` and `vo=gpu-next` to make the whole thing work
